src/nfa/operations.cc

```diff
@@ -1169,17 +1169,15 @@ Run mata::nfa::encode_word(const Alphabet* alphabet, const std::vector<std::stri
     return { .word = alphabet->translate_word(input) };
 }
 
-std::set<mata::Word> mata::nfa::Nfa::get_words(unsigned max_length) {
+std::set<mata::Word> mata::nfa::Nfa::get_words(unsigned max_length, const Symbol first_epsilon) {
     std::set<mata::Word> result;
 
     // contains a pair: a state s and the word with which we got to the state s
     std::vector<std::pair<State, mata::Word>> worklist;
     // initializing worklist
     for (State init_state : initial) {
-        worklist.push_back({init_state, {}});
-        if (final.contains(init_state)) {
-            result.insert(mata::Word());
-        }
+        worklist.emplace_back(init_state, Word{});
+        if (final.contains(init_state)) { result.insert(Word{}); }
```

Comment:
Is there a reason for putting this on one line?

Reply:
There is not. Just that it is shorter. A matter of taste change (caused by the fact that I earlier modified the body of the condition and later removed the added code again).